src/components/resource-picker/resource-picker-dialog.js

```diff
@@ -2,11 +2,9 @@ import React from 'react';
 import { Modal } from 'antd';
 import PropTypes from 'prop-types';
 import ResourceSelector from './resource-selector.js';
-import { STORAGE_LOCATION_TYPE } from '../../domain/constants.js';
+import { SOURCE_TYPE } from '../../domain/constants.js';
 
-// This can later be made configurable by individual plugins
-// (when we have more locations than just from the public/private CDN).
```

Comment:
It finally happened..happened..happened...

Reply:
:laughing: although I might be slightly mad for some time now